client/lib/domains/get-domain-type-text.js

```diff
@@ -6,19 +11,19 @@ import { type as domainTypes } from './constants';
 export function getDomainTypeText( domain = {} ) {
```

Comment:
Since we use the result this returns as is in some analytics calls ([example](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx#L303)), we probably should create a `getDomainTypeTextI18n()` function instead and use that, but I'm not a fan.

The other solution is to use `domain.type` instead in analytics calls, and call translate here as you're doing right now.

Reply:
yeah - domain.type in the analytics calls seems like the best approach